### DISCUSSION TITLE
redirection

### DISCUSSION THREAD

**Discussion by OlaniyiHope** [vercel/community#6416] [Help]
### Summary

my domains are not redirecting from https://mydomain.com to www......and i have used the 301 permanemty redirect but it is not still redirecting, what can i do

### Example

https://adzenon.com is not redirecting to https://www.adzenon.com

### Steps to Reproduce

you can just tell me what you di that worked for you

**Comment by versecafe** [comment]
@OlaniyiHope Is your redirect configured at the project level inside Vercel?

**Reply by versecafe** [reply]
<img width="1306" alt="image" src="https://github.com/vercel/community/assets/147033096/e9bd78a1-d5f9-4a0a-b14f-a926319a8f40">

**Comment by OlaniyiHope** [comment]
so should i do it like the way the picture you sent to me is

**Comment by OlaniyiHope** [comment]
yes
but it is not the non www i am trying to redirect to the www

**Comment by OlaniyiHope** [comment]
but it is the not www i want to redirect to the www....this is the picture
of what i have....[image: Screen Shot 2024-04-05 at 10.42.53 PM.png]

**Comment by OlaniyiHope** [comment]
that is what i did is that wrong....because it is not redirecting from
https://adzenon.com to https://www/adzenon.com

**Comment by versecafe** [comment]
@OlaniyiHope handle it at the domain level like shown above, currently whatever you are doing (as of an hour ago) infinitely redirects the user in circles causing hundreds of page reloads, just handle it at the project level and it becomes easy to manage and work with.

**Comment by OlaniyiHope** [comment]
but i cannot redirect here this is what i have, it is showing no redirect
possible....[image: Screen Shot 2024-04-06 at 6.36.54 AM.png]

**Comment by OlaniyiHope** [comment]
i dont get that....but i sent what i had in in the redirect on the www and
non www.....please do well to check and help me resolve it, the image is
attached to the second reply.thanks
